🐞Vector2 compare is always true

While playing the game, I'm running into:

Assets/Scripts/AI/SearchAStar.cs(73,39): warning CS0472: The result of comparing value type `UnityEngine.Vector2' with null is always `true'

--- a/Assets/Scripts/AI/SearchAStar.cs
+++ b/Assets/Scripts/AI/SearchAStar.cs
@@ -70,7 +70,7 @@ public class SearchAStar {
                 endLoc = new Vector2(con.Destination.x, con.Destination.y);
                 endCost = current.CostSoFar + con.Cost;
 
-                if (_debugMode && con.From != null) {
+                if (_debugMode && con.From != Vector2.zero) {
                     UnityEngine.Debug.DrawLine(con.From, con.Destination, Color.blue,2,false);
                 }
 

